ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/af-commons/v1/fieldTemplates/characterCounter.html

```diff
@@ -0,0 +1,6 @@
+<template data-sly-template.characterCounter="${@ componentId, showCharCounter, maxLength, bemBlock}">
+  <div data-sly-test="${showCharCounter}" class="${bemBlock}__charcounter">
+    <span class="${bemBlock}__charcount">0</span>
+    <span data-sly-test="${maxLength}" class="${bemBlock}__maxchar">/${maxLength}</span>
```

Comment:
digit 0 and maxLength will require localization as per locale

Reply:
Done